eng/validation/test-publishing.ps1

```diff
@@ -0,0 +1,66 @@
+Param(
+  [Parameter(Mandatory=$true)][string] $buildId,
+  [Parameter(Mandatory=$true)][string] $azdoToken,
+  [Parameter(Mandatory=$true)][string] $barToken
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference = 'Stop'
+
+. $PSScriptRoot\..\common\darc-init.ps1
+
+$global:buildId = $buildId
+$global:targetChannel = "General Testing"
+$global:azdoToken = $azdoToken
+$global:barToken = $barToken
+
+function Find-BuildInTargetChannel(
+    [string] $buildId,
+    [string] $targetChannelName
+)
+{
+    $build = darc get-build --id $buildId --azdev-pat $global:azdoToken --password $global:barToken
+
+    $index = 0
```

Comment:
You can pass `--output-format json` to get-build. It will return its response as json, and you will be able to get all this info a lot easier.

Reply:
Yeah, that was magical 😄 

![spongebob](https://lh3.googleusercontent.com/proxy/vOJarA0mJLwnVuKRFhgk_9IfO_hIREUh6vU3UMjlxfxYRr9B2RmW3fOZqnYexRufmEjK1QyyKcm3B90iDRVmUnYOvgDZj6m-a7jUQcOh_op8s1n9oRNsYCX18HypCAXB_rwNCGrB-vo6KWsNpJEHgm7iSt5ux_iiLlA)